scripts/lib/aws/elasticache.sh

```diff
@@ -12,6 +12,29 @@ SCRIPTS_DIR="$ROOT_DIR/scripts"
 # functions for tests
 #################################################
 
+# k8s_controller_apply_temp_creds adds AWS temporary credentials
+# to service controller running on kubernetes cluster.
+# it depends upon:
+#   $service_name
+#   $AWS_ACCESS_KEY_ID
+#   $AWS_SECRET_ACCESS_KEY
+#   $AWS_SESSION_TOKEN
+k8s_controller_apply_temp_creds() {
+  kubectl -n ack-system set env deployment/ack-"$service_name"-controller \
+    AWS_ACCESS_KEY_ID="$AWS_ACCESS_KEY_ID" \
+    AWS_SECRET_ACCESS_KEY="$AWS_SECRET_ACCESS_KEY" \
+    AWS_SESSION_TOKEN="$AWS_SESSION_TOKEN" 1>/dev/null
+}
+
+# reload_temp_credentials generates AWS temporary credentials
+# and adds them to service controller running on kubernetes cluster.
+reload_temp_credentials() {
+  echo -n "generating AWS temporary credentials and adding to env vars map ... "
+  aws_generate_temp_creds
+  k8s_controller_apply_temp_creds
+  sleep 15
```

Comment:
please add an:

```bash
echo "ok."
```

here since the `echo -n` on line 32 is used to prevent a newline...

Reply:
Updated script.